Some extra points

emmyft opened the pull request:
I don't think they are strictly necessary, but I think they have been helpful in the past especially for folks who are not used to hackathons/Sprints/coworking... please feel free to reword/ignore. Hope they help!

emmyft committed:
Added contributors' roles and tips on reminders

Please feel free to amend / reject if it's too much detail!

emmyft committed:
Add a tech check

emmyft committed:
clarify what i meant

tobyhodges commented on before.md:
```diff
@@ -13,6 +13,9 @@
   establish communications norms, and divide responsibilities. (See below for more details.)
+- **Consider participants' roles and expertise.** 
+  - Do you have a team with diverse skillsets and topic knowledge required to meet your high-level objectives? If not, consider finding and inviting other people.
+  - Consider the power dynamics in the group, e.g. if you have a PI and a early-career PhD candidate participating, how they can contribute effectively and comfortably.
```
This is a great point - thank you or raising it. Could you provide some more specific recommendations for this? E.g. how would you suggest to plan the sprint so that the hypothetical PI and PhD student you mention could both contribute effectively and comfortably?

tobyhodges commented on before.md:
```diff
@@ -32,6 +35,7 @@
     - Conversely, you may want to avoid planning the sprint that clashes with another event
     if your team is going to be engaged elsewhere or have other commitments.
 - **Consider promoting the sprint.** Generally, you might want to promote your upcoming sprint to attract other contributors.
+    - Consider whether you need a system to track participation for the Sprint, e.g. a Google Form/Eventbrite page
```
Similar to the comment above, I think an extra sentence here to describe how the organisers might decide whether tracking participation is necessary or not, would make this point complete. E.g. for in-person events it is often necessary to collect registrations to estimate numbers for booking a room/catering etc. But these are less relevant to virtual events. Under which circumstances do you think it would be helpful to track participation in advance?

tobyhodges reviewed (commented):
Thank you so much for adding these suggestions @emmyft 

I would love to see a little more information/specifics added to two of these. If you have time to look at my comments and add a few extra sentences, that would be really great.

emmyft committed:
Add examples for mitigating power dynamics

emmyft committed:
added more about systems for registration

emmyft commented:
Thanks @tobyhodges! Very useful feedback - I've added some more details to it. I'm noticing merge conflicts but not entirely sure what order you want to put things (maybe some of the things I've added don't make sense / are repeated now elsewhere in the guide?)... anyway, I'll leave things with you (:

tobyhodges commented:
Thank you @emmyft! I will get things organised and merged 👍 

@all-contributors please add @emmyft for review and code
